### PR TITLE
Jquery search

### PR DESCRIPTION
**Pull request by ritolia1**
Using this feature we can get the list of movie in a particular year and further can get detailed description of the specific movie. Each page show list of 20 movies and more movies can be viewed by toggling over the buttons(next and previous). If movie is searched without year , alert will be generated.
Movie details are fetched from 2 different Api.

### CODE REVIEW

File: public/css/movie-detail.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.image_bar{
+	margin-top: 20px;
```
Comment (shreedharshetty) [inline public/css/movie-detail.css]: combine in to margin: top right bottom left

Comment (ritolia1) [inline public/css/movie-detail.css]: Done with it.

**Comment by ritolia1** [comment]
Made changes as suggested. Kindly check the changes made.

File: config/config.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+exports.port = process.env.PORT || 8090
+exports.api_key= "306766094e835571482c61d778664850"
```
Comment (shreedharshetty) [inline config/config.js:2]: 👍

**Pull request merged**